app/models/vm_or_template.rb

```diff
@@ -846,7 +846,8 @@ def ems_host_list
     [ext_management_system, "ems", host, "host"].each_slice(2) do |ems, type|
       if ems
         params[type] = {
-          :address    => ems.address,
+          # TODO: (Julian) Removing instances of address. Hopfully this won't be missed.
```

Comment:
@juliancheal This method is used by SSA here [app/models/vm_scan.rb#L246](https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_scan.rb#L246) and here [app/models/vm_scan.rb#L453](https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_scan.rb#L453).  The data would be passed down to the scanning code.

@roliveri Do you know if that code relies on `:address` being passed?